test/db/rzil/bf

```diff
@@ -1,6 +1,6 @@
 NAME=hello world
 FILE=bins/bf/hello-ok.bf
-ARGS=-b32 -eio.cache=1
+ARGS=-eio.cache=1
```

Comment:
is even io.cache needed? i think not.

Reply:
currently it's needed ... it shouldn't be needed though

Reply:
It is needed for the emulation. The alternative is mapping a new malloc.

Reply:
Well if `oml` says the memory area is writable then i think it should be writable no matter the value of `io.cache`